src/tigerbeetle-java/src/main/java/com/tigerbeetle/Account.java

```diff
@@ -145,19 +145,18 @@ public void setLedger(int ledger) {
 
     void save(ByteBuffer ptr) {
 
-        ptr
-                .putLong(id.getLeastSignificantBits())
-                .putLong(id.getMostSignificantBits())
-                .putLong(userData.getLeastSignificantBits())
-                .putLong(userData.getMostSignificantBits())
-                .put(Struct.RESERVED)
-                .putInt(ledger)
-                .putShort(code)
-                .putShort(flags)
-                .putLong(debitsPending)
-                .putLong(debitsPosted)
-                .putLong(creditsPending)
-                .putLong(creditsPosted)
+        ptr.putLong(id.getLeastSignificantBits()) //
```

Comment:
The formatter wants all these empty comment lines? Never seen a format like this

Reply:
Yes, that's a common workaround to avoid the formatting messing with line breaks ... I think removing the fluent usage of this API is better.